examples/cli/discovery.md

```diff
@@ -49,34 +49,14 @@ You can see that the model is discovered, and some stats are printed, but no bui
 
 > See the [Analyze Only documentation](https://github.com/groq/mlagility/blob/main/tools_user_guide.md#analyze-only) for more details.
 
-## Search Directory
+## Benchmark Multiple Scripts
 
-The `--search-dir` argument to `benchit` (shorthand: `-s`) is useful for pointing `benchit` to look for the input script in a directory other than the current command line location.
+If you want to benchmark an entire corpus of models, but you don't want to call `benchit` individually on each model you may provide more than one python file to benchit at a time.
 
 For example, the command:
 
 ```
-benchit benchmark scripts/hello_world.py
-```
-
-is equivalent to the command:
-
-```
-benchit benchmark hello_world.py --search-dir scripts
-```
-
-The `--search-dir` argument is not very exciting on its own, however you can see the [Benchmark All](#benchmark-all) tutorial to see how it is powerful when combined with the `--all` argument.
-
-> See the [Search Directory documentation](https://github.com/groq/mlagility/blob/main/tools_user_guide.md#search-directory) for more details.
-
-## Benchmark All Scripts
-
-If you want to benchmark an entire corpus of models, but you don't want to call `benchit` individually on each model, then the `--all` argument is for you. `--all` will instruct `benchit` it iterate over every script in the search directory.
-
-For example, the command:
-
-```
-benchit benchmark --all --search-dir scripts
+benchit models/selftest/*.py
 ```
 
 Will iterate over every model in every script in the `scripts` directory, producing a result like this:
```

Comment:
You broke this tutorial since the command is pointing to `selftest` instead of `scripts`.

Reply:
I think this may have broken some of the other tutorials too, since they rely on the `scripts` dir being built. Why change this to `selftest`? Suggest reverting to `scripts`.

Reply:
Good catch!